manifest.toml

```diff
@@ -0,0 +1,100 @@
+#:schema https://raw.githubusercontent.com/YunoHost/apps/master/schemas/manifest.v2.schema.json
+
+packaging_format = 2
+
+id = "weblate"
+name = "Weblate"
+description.en = "Translation platform using Git and Python"
+description.fr = "Plateforme de traduction utilisant Git et Python"
+
+version = "5.3.1~ynh1"
+
+maintainers = ["YunoHost Contributors"]
```

Comment:
This should be the name of a person I think. Even better, a github username :) Or an empty array.

Reply:
Yes, as I said in the chat, I’m willing to take the maintainership here if needed, but I’ve no power here ^^